mapiproxy/servers/default/nspi/emsabp.c

```diff
@@ -633,6 +625,13 @@ _PUBLIC_ void *emsabp_query(TALLOC_CTX *mem_ctx, struct emsabp_context *emsabp_c
 		}
 		data = (void *) mvszA;
 		break;
+	case PT_LONG:
```

Comment:
@javierag: I don't see how this case can occur unless the ulPropTag passed to emsabp_query is just PT_LONG. I can't see any reasonably use case where this would happen. Am I missing something here?